packages/schema-type/package.json

```
@@ -14,19 +14,12 @@
     "./internal": "./dist/internal.js",
     "./deprecated": "./dist/deprecated.js"
   },
-  "typesVersions": {
-    "*": {
-      "index": [
-        "./src/typescript/exports/index.ts"
-      ],
-      "internal": [
-        "./src/typescript/exports/internal.ts"
-      ],
-      "deprecated": [
-        "./src/typescript/exports/deprecated.ts"
-      ]
-    }
-  },
+  "files": [
+    "dist",
+    "out",
+    "foundry.toml",
+    "cache"
```

This is a carry over from npmignore's line:
```
!cache/solidity-files-cache.json
```

But I wonder if we actually need this? cc @alvrs 

```suggestion
    "foundry.toml"
```

---

i don't remember why it was there, seems not very useful to include the files cache in the published packages 🤔